src/pages/iou/ReceiptSelector/index.native.js

```diff
@@ -94,13 +96,69 @@ function ReceiptSelector({route, report, iou, transactionID, isInTabNavigator})
     const [permissions, setPermissions] = useState('granted');
     const isAndroidBlockedPermissionRef = useRef(false);
     const appState = useRef(AppState.currentState);
-
+    const [isAttachmentInvalid, setIsAttachmentInvalid] = useState({
+        isInvalid: false,
+        title: '',
+        reason: '',
+    });
     const iouType = lodashGet(route, 'params.iouType', '');
     const reportID = lodashGet(route, 'params.reportID', '');
     const pageIndex = lodashGet(route, 'params.pageIndex', 1);
 
     const {translate} = useLocalize();
 
+    const hideInvalidAttachementModal = () => {
+        setIsAttachmentInvalid({
+            isInvalid: false,
+            title: '',
+            reason: '',
+        });
+    };
+
+    const showImageCorruptionAlert = useCallback(() => {
+        Alert.alert(translate('attachmentPicker.attachmentError'), translate('attachmentPicker.errorWhileSelectingCorruptedImage'));
```

Comment:
We have different functions for showing alerts for different cases in the file like `showPermissionsAlert`, `zshowGeneralAlert`. I think it would make sense to follow the convention here?